rabbitmq/datadog_checks/rabbitmq/data/conf.yaml.example

```diff
@@ -61,19 +61,26 @@ instances:
         ## @param include_aggregated_endpoint - boolean - optional - default: true
         ## Whether or not to include metrics from the aggregated endpoint (/metrics).
         ##
-        ## Note: This option must be set to true when 'unaggregated_endpoint' is unspecified.
+        ## You can learn more about this endpoint here:
+        ## https://www.rabbitmq.com/prometheus.html#default-endpoint
+        ##
+        ## Note: You must set this option to true if you do not specify the 'unaggregated_endpoint' option.
         #
         # include_aggregated_endpoint: true
 
         ## @param unaggregated_endpoint - string - optional - default: detailed
-        ## Which  unaggregated endpoint to use.
-        ## The choices are:
-        ##   - If the field is left unspecified, no endpoint is used.
-        ##   - "detailed" means the /metrics/detailed endpoint is used.
-        ##     You can also provide a query for this option, exactly as if you were making an HTTP request
-        ##     to this endpoint. For example, `detailed?family=queue_consumer_count`.
-        ##     You can specify this and the aggregated endpoint. In that case any duplicate metrics will
-        ##     be taken from this endpoint and the corresponding aggregated ones dropped.
+        ## Specify an endpoint without aggregation to collect more detailed metrics.
+        ##
+        ## For example, you can collect metrics tagged by queue or by exchange.
+        ## These metrics are served from the /metrics/detailed endpoint.
+        ## You can learn more about it here:
+        ## https://www.rabbitmq.com/prometheus.html#detailed-endpoint
+        ##
+        ## For brevity, you only need to specify "detailed" to start using this endpoint.
+        ## You can also provide a query exactly as if you were making an HTTP request
+        ## to this endpoint. For example, `detailed?family=queue_consumer_count`.
+        ##
+        ## You can specify both this and the aggregated endpoint. If you specify both endpoints, metrics from the unaggregated endpoint are prioritized over metrics from the aggregated endpoint.
```

Comment:
```suggestion
        ## You can specify both this and the aggregated endpoint. If you specify both endpoints, metrics from the unaggregated endpoint are prioritized over metrics from the aggregated endpoint in case of name clashes.
```